docker-compose.yml

```diff
@@ -33,7 +33,7 @@ services:
                 condition: service_healthy
 
     irods-client-http-api:
-        image: irods/irods_http_api:latest
+        image: irods/irods_http_api:0.2.0
```

Comment:
this will pin our docker-compose to 0.2.0 - not sure this is what we want.

latest is also currently an alias for 0.2.0 - so a rebuild will get the newest image...

Reply:
I figured it was better to pin it so it never breaks. This should help with debugging and users familiar with docker compose can assert/report what they're running.

I think everything in the compose file should be pinned to guard against unexpected behavior.

Thoughts?

Reply:
A fair policy.   Means we’ll have to touch this repo more.   But that’s not a bad thing.